src/htmlrenderer.cpp

```diff
@@ -1030,22 +1008,29 @@ void HtmlRenderer::add_media_link(std::string& curline,
 	const std::string& media_url, const std::string& media_title,
 	unsigned int media_count, LinkType type)
 {
+	std::string link_url;
```

Comment:
It's better to declare variables close to their use-site, to make the code slightly easier to understand ;)